openvinotoolkit/cvat
# Redirect query param

### Motivation and context


This PR provide redirect parameter for auth page for redirecting after sucessfull login.

### How has this been tested?


Manually

### Test scenarios

1. Create some task if not existed
2. Logout and go to ``/tasks/<task_id>`` it will redirect to ``/auth/login?next=/tasks/<task_id>``
3. Login
4. Check correct task is opened

Using token login add parameter next manually: ``/login-with-token/:sessionId/:csrfToken?next=/tasks/<task_id>``

### Checklist

~~- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
~~- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```
